File: ChatTalk.WebServer/Data/Entities/UserEntity.cs
namespace ChatTalk.WebServer.Data.Entities
{
    public class UserEntity
    {
        public long Id { get; set; }
        public string LoginId { get; set; } = string.Empty;
        public string Password { get; set; } = string.Empty;
        public string UserName { get; set; } = string.Empty;
        public DateTime CreateAt { get; set; } = DateTime.Now;
    }
}

File: ChatTalk.AuthServer/src/main/java/com/chattalk/auth/legacy/entity/UserEntity.java
package com.chattalk.auth.legacy.entity;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter
@Setter
@Entity
@Table(name = "users")
public class UserEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String userId;
    private String password;
    private String userName;
    private LocalDateTime createAt = LocalDateTime.now();
}
